spec/requests/users_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET #index' do
+    it 'includes correct placeholder text in the response body' do
+      get users_path
+      expect(response.body).to include('Hello! this is the user index')
+    end
+  end
+
+  describe 'GET #show' do
+    let(:user) { create(:user) }
+
+    it 'includes correct placeholder text in the response body' do
+      get user_path(user)
+      expect(response.body).to include('Hello! this is the user show')
+    end
+  end
+end
```

Comment:
- [ ] Kindly add tests fro the User's `show` and `index` actions for the controllers that ensure that the correct view template is rendered. Like you have done for the post_spec file as per the requirement.
`Example : expect(response).to render_template(:show)`